src/sentry/static/sentry/app/components/events/meta/annotatedText.tsx

```diff
@@ -107,26 +112,24 @@ function renderErrors(errors) {
   );
 }
 
-const AnnotatedText = ({value, chunks, errors, remarks, ...props}) => (
-  <span {...props}>
-    {renderValue(value, chunks, errors, remarks)} {renderErrors(errors)}
-  </span>
-);
-
-AnnotatedText.propTypes = {
-  value: PropTypes.node,
-  chunks: PropTypes.array,
-  errors: PropTypes.array,
-  remarks: PropTypes.array,
-};
-
-AnnotatedText.defaultProps = {
-  chunks: [],
-  errors: [],
-  remarks: [],
-};
+class AnnotatedText extends React.Component<Props, {}> {
+  render() {
+    const {value, meta, ...props} = this.props;
+    let component = renderValue(value, meta);
+    if (meta?.err) {
+      component = (
```

Comment:
```suggestion
   return (
```

Reply:
this code no longer exists because the tests started breaking